twisted/mail/scripts/mailmail.py

```diff
@@ -260,7 +262,9 @@ def loadConfig(path):
                         try:
                             id = int(id)
                         except ValueError:
-                            log("Illegal %sID in [%s] section: %s", section[0].upper(), section, id)
+                            log.error(
+                                "Illegal {}ID in [{}] section: {}".format(
+                                section[0].upper(), section, id))
```

Comment:
Please use the structured log call.

ex

```
log.info(
    "{request.uri}: status={response.status}, bytes={response.size}, etc...",
    request=request, response=response
)
```

For more details see 'The basics' section from the logger docs https://twistedmatrix.com/documents/current/core/howto/logger.html#the-basics

Already serialized strings for the logging systesm are so '90 :)